cmd/settlement/uphold.go

```diff
@@ -248,14 +250,42 @@ func UpholdUpload(
 			logger.Panic().Err(err).Msg("failed to record progress")
 		}
 
-		if !settlementTransaction.IsComplete() && !settlementTransaction.IsFailed() {
+		// We will later attempt to resolve all processing to complete or failed
+		if settlementTransaction.IsProcessing() {
+			someProcessing = true
+		} else if !settlementTransaction.IsComplete() && !settlementTransaction.IsFailed() {
 			allFinalized = false
 		}
```

Comment:
Should we have a final `else` here to cover any cases other than `complete` `processing`, or `failed`? Maybe with a loud log?

Reply:
I think this should cover all states:
processing -> set someProcessing to true
complete / failed -> do nothing
else -> set allFinalized to false